nengo_viz/swi.py

```diff
@@ -465,7 +465,7 @@ def handle_error(self, request, client_address):
                 exc_value.args[0] == errno.EBADF):
             return  # Probably caused by a server shutdown
         else:
-            print exc_type, exc_value, dir(exc_value)
+            print(exc_type, exc_value, dir(exc_value))
```

Comment:
We don't do that anywhere else, and it's only needed if someone is using Python 2.5 or earlier.